### DISCUSSION TITLE
Stop remembering application group

### DISCUSSION THREAD

**Discussion by gruvw** [qtile/qtile#4455] [Q&A]
Hello, I don't know how to stop the fact that a window will be reopened in the group it was last closed instead of the current group I am standing when opening the GUI program.

For example, I am in group "1", I open my web browser, then I close it (still in group "1").  
Then I go to group "2", I execute a shortcut to `lazy.spawn` my web browser and it spawns in group "1" when I wanted it to spawn in current group: group "2".

I hope this is clear. Thx

**Comment by gruvw** [comment]
The windows remember their group/workspace position across machine reboot so it must happen somewhere but I can't find a way to disable it...

**Reply by m-col** [reply]
Does this happen with all programs or just some/one? Some programs will remember it themselves and make their new windows open on a group that they choose. 

Make sure you don't have any `matches` passed to any `Group` instances interferring with this too, maybe that could be why

**Reply by gruvw** [reply]
No it happens with every GUI program. I don't have matches. With the same config, it never happened on my other PC.

**Reply by m-col** [reply]
What version are you using? If you're not on the git version, could you try version 0.23.0?
